### DISCUSSION TITLE
Be Patient

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#122] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Please be patient. It actually take time to code, and you all yelling at the owners  just demotivates them. Just wait.

**Comment by ghost** [comment, marked as answer]
FOR REAL!

**Reply by JelloFruitAlt** [reply]
don't worry, its just annoying little kids who can't code for themselves so they beg for you guys to do it.

**Reply by 0PLUSXIX** [reply]
hey guys, can you please hurry up and fix it because my teacher keeps closing out my tabs its so annoying 🤓🤓🤓

**Reply by 3kh0** [reply]
@0PLUSXIX really?

**Reply by The-Man-Is-Me** [reply]
That's funny.
